docs/openapi/schemas-pn-notification-v1.yaml

```diff
@@ -96,9 +109,11 @@ components:
         senderDenomination:
           description: Denominazione dell'ente mittente
           type: string
+          $ref: '#/components/schemas/Denomination'
         senderTaxId:
           description: Codice Fiscale dell'ente mittente
           type: string
+          $ref: '#/components/schemas/TaxId'
```

Comment:
Il codice fiscale di un ente può essere solo la P.Iva?

Reply:
da documentazione su ADE sembrerebbe di si. https://www.agenziaentrate.gov.it/portale/web/guest/schede/istanze/codice-fiscale-modello-aa5_6/scheda-info-cf-aa5_6-enti-e-pa

Reply:
anche utilizzando la ricerca dell'ente su https://www.indicepa.gov.it/ipa-portale/consultazione/indirizzo-sede/ricerca-ente la ricerca per codice fiscale ammette solamente 11 cifre

Reply:
Quindi potremmo limitare alle sole P.IVA?

Reply:
ok... limitato alle sole P.IVA